src/main/java/com/iab/gdpr/VendorConsent.java

```diff
@@ -282,6 +285,15 @@ public String getBinaryString() {
 
     }
 
+    /**
+     *
+     * @return an integer equivalent of allowed purpose id bits according to this consent string
+     */
+    public int getAllowedPurposesBits() {
```

Comment:
I think we can just use `return bits.getInt(PURPOSES_OFFSET, PURPOSES_SIZE);`

Reply:
of course! fixed.

Reply:
Ačiū !